libdazzle.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "libdazzle",
```

Comment:
Please put this module on the main one as there's no reason for splitting modules.